public/wallet-service-worker.mjs

```
@@ -10990,26 +10998,26 @@ function lp() {
     });
   }(Do)), Do.exports;
 }
-var dp = lp();
-const Hs = /* @__PURE__ */ Ud(dp), ru = Symbol.for("Dexie"), ii = globalThis[ru] || (globalThis[ru] = Hs);
+var hp = dp();
+const Hs = /* @__PURE__ */ Ud(hp), ru = Symbol.for("Dexie"), ii = globalThis[ru] || (globalThis[ru] = Hs);
```

ESLint

[error] Strings must use singlequote.

(@typescript-eslint/quotes)